### PR TITLE
In local mode, give an option to specify username, default to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
#2602 #2615 

Seems I cannot recover the previous key... Created another pair, updated the secret, and added the same key to user jenkins, wait for a next green run for the gce-reboot-1.4 job.

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -445,8 +449,7 @@ def create_parser():
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
     parser.add_argument(
-        '--multiple-federations', default=False, action='store_true',
-        help='If we need to run multiple federation control planes in parallel')
+        '--user', default='prow', help='Username used for ssh in local mode')
```
Comment (fejta) [inline scenarios/kubernetes_e2e.py]: This should not be a flag

Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: In case user uses their own ssh key?

Comment (fejta) [inline scenarios/kubernetes_e2e.py]: This shouldn't be set by the scenario. It should be set by the prow job definition or else the image -- probably the prow job.

If the user runs the scenario they will already have a USER var set correctly

Comment (fejta) [inline scenarios/kubernetes_e2e.py]: The only thing that is wrong before is that the user was root and so when we ran config-ssh we uploaded the ssh key for user root, but the VM does not allow user root to ssh

**Comment by krzyzacy** [comment]
defer to #2618

**Pull request closed without merging**